chrome/script.browserify.js

```diff
@@ -195,14 +195,19 @@ function fillLoginForm(login) {
 	}
 
 	update(field('input[type=password]'), ${JSON.stringify(login.p)});
-	update(field('input[type=email], input[type=text], input:first-of-type'), ${JSON.stringify(login.u)});
+	// Guess username field
+	var usernameFields = field('input[id=username], input[id=login], input[id=email]');
```

Comment:
could you explain that a little? in my example I excluded the `first-of-type` and rearranged fields, so I think the two code snippets will be identical... or not?